src/funding/funding.ts

```diff
@@ -340,18 +331,40 @@ export class Funding {
     this.accountBalance = await contract.balanceOf(this.ethereumService.defaultAccountAddress);
   }
 
-  //moving this getter locally for the proposal lead
+  public async setFundingTokenAllowance(): Promise<void> {
+    const contract = this.tokenService.getTokenContract(this.firstDaoTokens[this.selectedToken].address);
+    this.userFundingTokenAllowance = await contract.allowance(this.ethereumService.defaultAccountAddress, this.deal.daoDepositContracts.get(this.firstDao).address);
+  }
+
+  @computedFrom("ethereumService.defaultAccountAddress", "deal.daoRepresentedByCurrentAccount", "deal.primaryDao")
   public get firstDao() : IDAO{
     return this.deal.isRepresentativeUser ? this.deal.daoRepresentedByCurrentAccount : this.deal.primaryDao;
   }
 
-  //moving this getter locally for the proposal lead
+  @computedFrom("ethereumService.defaultAccountAddress", "deal.daoRepresentedByCurrentAccount", "deal.primaryDao")
```

Comment:
these look like the wrong values to observe

Reply:
You're right. I adjusted them. Thanks!